salt/states/win_wusa.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+'''
+Microsoft Updates (KB) Management
+
+This module provides the ability to enforce KB installations
+from files (.msu), without WSUS.
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
+import logging
+
+# Import salt libs
+import salt.utils.platform
+import salt.exceptions
+
+log = logging.getLogger(__name__)
+
+
+# Define the module's virtual name
+__virtualname__ = 'win_wusa'
+
+
+def __virtual__():
+    '''
+    Load only on Windows
+    '''
+    if not salt.utils.platform.is_windows():
+        return False, 'Only available on Windows systems'
+
+    return __virtualname__
+
+
+def installed(name, source):
+    '''
+    Enforce the installed state of a KB
+
+    name
+        Name of the Windows KB ("KB123456")
+    source
+        Source of .msu file corresponding to the KB
+
+    '''
+    ret = {
+        'name': name,
+        'changes': {},
+        'result': False,
+        'comment': '',
+        'pchanges': {},
```

Comment:
Since we haven't ironed out the details behind `pchanges` we're in the process of removing it. Please just use `changes`.

Reply:
Done.